### ISSUE TITLE
Bug Report `color.rgba2rgb` assume all rgba values are in range [0, 1] but they are in [0, 255]

### ISSUE THREAD

**Issue by wenmin-wu** [scikit-image/scikit-image#3191]
## Description
I used the `color.rgba2rgb` to transform my image, but didn't get the desired result. So, I scanned the implementation and found the author assume all rgba values are in range [0, 1], but they are in [0, 255]

## Way to reproduce
You can reproduce it by following code, suppose you have installed the `Pillow` and use the notebook which is easy for image displaying.
```
from PIL import Image as pil_image
from IPython.display import display
%matplotlib inline
from skimage.color import rgba2rgb
from skimage import data
img_rgba = data.logo()
display(pil_image.fromarray(img_rgba), 'RGBA')
img_rgb = rgba2rgb(img_rgba)
display(pil_image.fromarray(img_rgb.astype(np.uint8), 'RGB'))
```
the img_rgba is displayed as:

![wx20180613-134341](https://user-images.githubusercontent.com/9409333/41332293-da7ec898-6f0f-11e8-9722-7bbaec7eae33.png)

and the img_rgb is displayed as:
![wx20180613-134354](https://user-images.githubusercontent.com/9409333/41332311-ec616b74-6f0f-11e8-92f3-64a9d62aad38.png)

the os is ubuntu, and the python version is 2.7, the skimage version is 0.14.0

**Comment by jni** [comment]
@wenmin-wu This is actually expected behaviour, please see http://scikit-image.org/docs/dev/user_guide/data_types.html

You should replace `image.astype(np.uint8)` with `from skimage import img_as_ubyte` then `img_as_ubyte(image)`.

We are aware that this convention can be confusing and are discussing ways to fix this in #3009. So, in the meantime I will close this issue as duplicate, please feel free to continue the discussion in #3009.

**Comment by wenmin-wu** [comment]
@jni Thx for your reply. I replaced the code as you suggested and got the desired result.
